Remove message after course type selection

awaisdar001 opened the pull request:
[ECOM-3150 - CAT "select a course type" remains after selected](https://openedx.atlassian.net/browse/ECOM-3150)

Issue
-
The "select a course type" message remains on the page after the course type has been selected.
Acceptance Criteria
-
VERIFY that the "select a course type" message is removed upon selection of a course type.

clintonb commented on acceptance_tests/test_course_create.py:
```diff
@@ -0,0 +1,20 @@
+"""
```
This should be a Jasmine test.

awaisdar001 replied:
Ok got it.

awaisdar001 commented:
@aamir-khan Please review.

aamir-khan commented:
:+1: Looking good to me.

awaisdar001 commented:
@adampalay  Please review.

adampalay commented on ecommerce/static/js/test/specs/views/course_create_view_spec.js:
```diff
@@ -25,6 +25,14 @@ define([
             });
+            it('select a course type message is removed', function () {
+                expect(view.$el.find('.course-seat.empty').hasClass('hidden')).toBe(false);
+                expect(view.$el.find('.course-types input[type=radio]').length).toBe(4);
```
isn't what you want to check here that there are no checked radio buttons?

adampalay replied:
and do you want `toBe` or `toEqual`?

awaisdar001 commented:
@adampalay  Updated the PR and addressed your comments. 
* Using `toEqual` rather than `toBe`
* Checking no checkboxes when the view is rendered.

adampalay commented on ecommerce/static/js/test/specs/views/course_create_view_spec.js:
```diff
@@ -25,6 +25,15 @@ define([
+
+                view.model.set('type', 'credit');
+                expect(view.$el.find('.course-types input[type=radio]:checked').length).toEqual(1);
+                expect(view.$el.find('.course-seat.empty').hasClass('hidden')).toEqual(true);
```
sorry to be a stickler, but I'm guessing you want this one to be `toBe` — equal for numbers, "be" for booleans

awaisdar001 replied:
Understandable. I will update the PR and merge.

adampalay commented:
:+1:

awaisdar001 commented:
@adampalay Also updating the `AUTHORS` file.

AlasdairSwan commented:
Looks good :+1:
